integration_testing/features/pressing-the-information-icon.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Pressing the information icon
+
+	Background:
+		Given I am choosing a destination to move content into
+			And I click the information icon on a particular topic or content list item
+
+	Scenario: Pressing the information icon
+		When I click the information icon
+		Then a side sheet on a scrim should appear sliding out from the righ side of the screen

```

Comment:
Maybe `Then a sidebar opens from the right side of the screen` would sound better.